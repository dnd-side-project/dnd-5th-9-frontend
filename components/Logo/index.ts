export { default as Spacing } from "./Spacing";
export { default as Symbol } from "./Symbol";
export { default as Wordmark } from "./Wordmark";
