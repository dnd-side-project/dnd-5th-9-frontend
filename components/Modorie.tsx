import React from "react";

const Modorie = () => {
  return <div>도리도리 모도리</div>;
};

export default Modorie;
