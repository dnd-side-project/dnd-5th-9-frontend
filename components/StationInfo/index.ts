export { default as StationInfo } from "./StationInfo";
