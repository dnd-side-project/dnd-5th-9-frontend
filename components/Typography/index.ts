import { TypoStyle } from "./Typography.styled";

export { default as Typography } from "./Typography";
export { TypoStyle };
