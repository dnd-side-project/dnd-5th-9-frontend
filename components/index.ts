export * from "./Icon";
export * from "./Input";
export * from "./StationInfo";
export * from "./Typography";
