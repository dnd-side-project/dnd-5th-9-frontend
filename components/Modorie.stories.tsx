import React from "react";

import Modorie from "./";

export default {
  title: "Components/Modorie",
  component: Modorie,
};

export const Standard = () => <Modorie />;
