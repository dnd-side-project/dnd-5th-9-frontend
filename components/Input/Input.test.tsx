import { render } from "@testing-library/react";
import { Input } from "@/components";

it("matches snapshot", () => {
  const utils = render(<Input />);
  expect(utils.container).toMatchSnapshot();
});
