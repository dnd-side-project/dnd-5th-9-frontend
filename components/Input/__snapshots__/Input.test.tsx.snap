// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div>
    <div
      class="sc-gsDKAQ fMoos"
    >
      <input
        class="sc-dkPtRN fqbiRB"
        placeholder="Placeholder"
        required=""
        type="text"
      />
    </div>
  </div>
</div>
`;
